### ISSUE TITLE
Error compiling

### ISSUE THREAD

**Issue by vrajput** [adafruit/Adafruit_FONA_Library#28]
Just downloaded FONA_Library, 
Getting error in compilation

In member funtion 'boolean Adafruit_FONA::TCPsend(char*, uint_t);
![image](https://cloud.githubusercontent.com/assets/75521/8313181/3039ba90-19af-11e5-94a0-21937e1d12c7.png)

**Comment by jkeefe** [comment]
Me, too. Running Arduino IDE 1.0.5 on a Mac. Same error, same line.

**Comment by ladyada** [comment]
Hiya, can you try 1.6.4?

**Comment by toddtreece** [comment]
It looks like this is related to older versions of the IDE. The issue is fixed in v1.0.6+ of the IDE, so upgrading to the latest version (v1.6.5) should fix the compile errors you are seeing.

![screen shot 2015-06-24 at 10 46 57 am](https://cloud.githubusercontent.com/assets/360020/8333098/6826ddba-1a5f-11e5-81ca-14857a04e943.png)

Relevant changes from the Arduino IDE changelog:

```
ARDUINO 1.0.6 - 2014.09.16

[core]
* avr: Improved USB-CDC write speed (Justin Rajewski)
* avr: Improved USB-CDC read code (Paul Brook)
* avr: Fixed race condition in USB-CDC transmit (Paul Brook)
* avr: Added replacement stub for cstdlib atexit() funciton (Christopher Andrews)
* Fixed wrong NULL pointer handling in Stream class (Amulya Kumar Sahoo)
* Backported String class from IDE 1.5.x (Matt Jenkins)
* Backported Print class from IDE 1.5.x
* Backported digitalPinToInterrupt macro from IDE 1.5.x
* Added initVariant() hook to allow 3rd party variant-specific initialization
* Added replacement stub for cstdlib atexit() function (Christopher Andrews)
```

**Comment by jkeefe** [comment]
Thanks! Will try that!
